server/test/instant/db/indexing_jobs_test.clj

```diff
@@ -257,11 +389,12 @@
                             (every? (fn [{:keys [id]}]
                                       (= "completed" (:job_status (jobs/get-by-id id))))
                                     [job]))
-                          1000)
+                          5000)
```

Comment:
Had to bump this timeout--I think our tests instance runs into noisy neighbor problems against our other test instances.